### ISSUE TITLE
--coef_values file

### ISSUE THREAD

**Issue by JackDelasoudas** [m-bone/Bond_React_Python#9]
hello i try to simulate bond breaking not bond forming. So I have two data files generated by moltemplate that contain the connectivity and the information about the potential coefficients on each state (I attach them). My question is when I run the AutoMapper.py . clean command which file of the system.in.settings sould i attach. The one of the pre or of the post reacrion. Note that there are different types of monds in each situation cause bonds are deleted. 
Thank you!
[reaction_files.zip](https://github.com/m-bone/AutoMapper/files/12049734/reaction_files.zip)

**Comment by m-bone** [comment]
Hi thanks for the good question! This would be easiest fixed if you are using Moltemplate, I would suggest not cleaning the system.in.settings file using cleanmoltemplate.sh. This means the system.in.settings file has all coeffs and AutoMapper clean will choose the ones needed for the simulation. You're right that neither of your current files will work, and the only way to fix them would be to combine them by hand. Best of luck!

**Comment by JackDelasoudas** [comment]
Thanks!

**Comment by JackDelasoudas** [comment]
Have you ever simulated bond breaking with the fix bond/react command??

**Comment by m-bone** [comment]
Yes however only when also making a bond as in a polymerisation reaction. I've not personally tried just bond breaking, and if I'm honest I'm not sure how you would only break a bond as bond/react is typically checking for cutoff distances to make a change to the topology. If you're just breaking a bond, presumably the atoms are quite close so a distance check will always pass? Are you using a different check?

**Comment by JackDelasoudas** [comment]
I use a min max criterion based on the tension distance of the bond. However I struggle with something else. I use dreiding force field that includes dihedrals and impropers. When i generate the files in AutoMapper the generated files that are reduced to the minimum number of atoms for the reaction do not include the dihedrals and the impropers and by that i mean that they have edge atoms that are realy close to the bonds that break. how can i solve it?

**Comment by JackDelasoudas** [comment]
And I get this error,

ERROR: Fix bond/react: Bond type affected by reaction rxn1 is too close to template edge (src/REACTION/fix_bond_react.cpp:2653)                                                                                                              Last command: fix               fxrct all bond/react stabilization yes statted_grp .03           react rxn1 all 100 0.0 10.0 mol1 mol2 map.txt stabilize_steps 100

**Comment by JackDelasoudas** [comment]
[fix.zip](https://github.com/m-bone/AutoMapper/files/12207672/fix.zip)
I get this traceback when I try to use map function i these files. The conected atoms are 17 and 72. 

The error is 

AssertionError: Different numbers of atoms have been found in the pre- and post-bond partial structures.
